doc/ref_arch/openstack/chapters/chapter04.md

```diff
@@ -594,16 +581,6 @@ Heat is the orchestration service using template to provision cloud resources, H
 #### 4.3.1.9 Horizon
 Horizon is the Web User Interface to all OpenStack services. Horizon has services running on the control nodes and no services running on the compute nodes.
 
-<!--
-#### 4.3.1.10 Cyborg
```

Comment:
Keep commented the content about Cyborg, as it may be reintroduced if we move to a higher OpenStack release